app/controllers/api/guidance_controller.rb

```diff
@@ -16,18 +17,32 @@ def release_notes
     end
 
     def page
-      path = params[:page]
-
-      template = {
-        "swagger-api-documentation" => "swagger_api_documentation",
-        "guidance-for-lead-providers" => "guidance_for_lead_providers",
-        "sandbox" => "sandbox",
-        "guidance-for-lead-providers/api-data-states" => "guidance_for_lead_providers/api_data_states",
-        "guidance-for-lead-providers/data-syncing" => "guidance_for_lead_providers/data_syncing",
-        "guidance-for-lead-providers/ids-explained" => "guidance_for_lead_providers/ids_explained",
-      }.fetch(path)
-
-      render "api/guidance/" + template
+      template = "api/guidance/#{params[:page].underscore}"
+
+      if template_exists?(template)
+        render template
+      else
+        render 'errors/not_found', status: :not_found
+      end
+    end
+
+  private
+
+    def guidance_pages
+      [
+        { title: "API IDs explained", path: "guidance-for-lead-providers/ids-explained" },
```

Comment:
should we update IDs explained title to include API everywhere so it's consistent, one to ask Mark

Reply:
confirmed with mark will change it